app/views/order.py

```diff
@@ -93,7 +101,8 @@ def order_item_create(id):
     if current_user.is_anonymous() and not current_order.public:
         flash('Please login to see this order.', 'info')
         abort(401)
-    form = AnonOrderItemForm() if current_user.is_anonymous() else OrderItemForm()
+    form = AnonOrderItemForm() if current_user.is_anonymous(
```

Comment:
one line?

Reply:
Luckily we don't have to spend even one line on code style discussions anymore. Whatever black does is correct :)